app/src/main/res/values/strings.xml

```diff
@@ -20,7 +20,8 @@
     <string name="learning_cat_0_descr">red</string>
     <string name="learning_cat_1_descr">green</string>
     <string name="learning_cat_2_descr">blue</string>
-    <string name="learning_settings_tooltip_presentation">Play the game in presentation mode: an image must be classified based on an exact copy</string>
+    <string name="learning_settings_tooltip_presentation">Play the game in presentation mode: an image must be classified against on an exact copy</string>
```

Comment:
```suggestion
    <string name="learning_settings_tooltip_presentation">Learn in presentation mode: an image must be classified against an exact copy</string>
```

Reply:
Damnit I forgot again haha, sorry about that.